docs/modules/prompts/prompt_templates/getting_started.md

```diff
@@ -69,30 +63,81 @@ multiple_input_prompt.format(adjective="funny", content="chickens")
 # -> "Tell me a funny joke about chickens."
 ```
 
+If you do not wish to specify `input_variables` manually, you can also create a `PromptTemplate` using `from_templates` class method. `langchain` will automatically infer the `input_variables` based on the `template` passed.
```

Comment:
@hwchase17 thoughts on putting this section first? It's the least cognitive load. Then we can add information about configurability later.

Reply:
i agree!